test/minideflate.c

```diff
@@ -13,16 +13,18 @@
 #if defined(_WIN32) || defined(__CYGWIN__)
 #  include <fcntl.h>
 #  include <io.h>
-#  include <string.h>
 #  define SET_BINARY_MODE(file) setmode(fileno(file), O_BINARY)
-#  ifdef _MSC_VER
-#    define strcasecmp _stricmp
-#  endif
 #else
-#  include <strings.h>
 #  define SET_BINARY_MODE(file)
 #endif
 
+#ifdef _MSC_VER
```

Comment:
It is possible to compile using LLVM against Windows SDK. In this instance, shouldn't it still be defining `strcasecmp`?

Reply:
clang-cl doesn't define `_MSC_VER` by default.

Reply:
Then it should be checking `_WIN32` instead of `_MSC_VER`.